Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:3.0.2-alpine
+RUN apk update
+RUN apk add --virtual build-dependencies build-base ruby-dev libpq postgresql-dev tzdata
+WORKDIR /code
+COPY Gemfile /code
+RUN bundle config set --local without 'development test'
```

Comment:
I'm really nitpicking now, but you can put `WORKDIR` and `bundle config` to the very top, as those layers will likely never change